x-pack/plugins/uptime/public/components/overview/monitor_list/filter_status_button.test.tsx

```diff
@@ -7,7 +7,8 @@
 
 import React from 'react';
 import { FilterStatusButton, FilterStatusButtonProps } from './filter_status_button';
-import { renderWithRouter, shallowWithRouter, MountWithReduxProvider } from '../../../lib';
+import { renderWithRouter, shallowWithRouter } from '../../../lib/helper/enzyme_helpers';
+import { MountWithReduxProvider } from '../../../lib';
```

Comment:
i think dist size didn't improve, probably because here it's still doing imporrt from /lib